lib/decorators/next-steps.js

```diff
@@ -11,6 +11,9 @@ module.exports = settings => {
 
     return getNextStepsForUser(c, user.profile)
       .then(nextSteps => {
+        // recovered should not appear in next steps radios
+        nextSteps = nextSteps.filter(s => s.id !== recovered.id);
```

Comment:
We don't do this for `discardedByAsru` which is added to the available steps in the same way and displays a similar UI - do we need it?

Reply:
yes or no, I do not know how that works